includes/blocks/course-theme/class-course-progress-counter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * File containing the Course_Progress class.
+ *
+ * @package sensei
+ * @since
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * Class Course_Progress is responsible for rendering the '< Prev Lesson | Next Lesson >' blocks.
+ */
+class Course_Progress_Counter {
+	/**
+	 * Course_Progress constructor.
+	 */
+	public function __construct() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/course-theme-course-progress-counter',
+			[
+				'render_callback' => [ $this, 'render' ],
+			]
+		);
+	}
+
+	/**
+	 * Renders the block.
+	 *
+	 * @access private
+	 *
+	 * @return string The block HTML.
+	 */
+	public function render() : string {
+		$course_id = \Sensei_Utils::get_current_course();
```

Comment:
Oh, it made me notice that I was rewriting this logic in another class. I raised [a PR](https://github.com/Automattic/sensei/pull/4436) to use this function too. =)